fcitx.el

```diff
@@ -577,6 +583,28 @@ Re-run the setup function after `fcitx' is started.")))
                        "GetCurrentState")
      2))
 
+(defun fcitx--activate-dbusv5 ()
```

Comment:
I'll suggest adding a separator to make the name easier to read:
```suggestion
(defun fcitx--activate-dbus-v5 ()
```

Please also update the others accordingly

Reply:
done